### PR TITLE
New package : python-dlib-19.20.0

### PR DESCRIPTION
**Pull request by pandom79**
I built dlib as python module. 
I copied from existent dlib package.
I'm ready for eventual corrections.
Regards

### CODE REVIEW

File: srcpkgs/python-dlib/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'python-dlib'
+pkgname=python-dlib
+version=19.20.0
+revision=1
+wrksrc=dlib-${version}
+build_style=python-module
```
Comment (sgn) [inline srcpkgs/python-dlib/template:6]: We don't accept new python 2 package (there're exception, but it's rare).

File: srcpkgs/python-dlib/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+hostmakedepends="cmake python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel
+			 $(vopt_if openblas openblas-devel)
```
Comment (sgn) [inline srcpkgs/python-dlib/template:9]: This is over indented. IOW, move left 2 TABs

File: srcpkgs/python-dlib/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+			 $(vopt_if x11 libX11-devel)"
+short_desc="C++ toolkit containing machine learning algorithms and tools (Python2)"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="BSL-1.0"
+homepage="https://pypi.org/project/dlib/"
+distfiles="${PYPI_SITE}/d/dlib/dlib-${version}.tar.gz"
+checksum=a3f5070df590c3f510dde2f4c84b1b20e3ef64a02932ed657da490834ecac7e4
+build_options="openblas x11"
+desc_option_openblas="Enable OpenBlas for performance"
+desc_option_x11="Enable X11 for GUI components"
```
Comment (sgn) [inline srcpkgs/python-dlib/template:19]: option_x11 has a common description, remove this line (but keep `_openblas`

Comment (pandom79) [reply srcpkgs/python-dlib/template]: Done. Now it's just python3.

**Comment by sgn** [comment]
On 2020-07-11 09:46:05-0700, pandom79 <notifications@github.com> wrote:
> Done. Now it's just python3.

Rename the package to python3-dlib too.
(I have neither built nor verify this PR).

-- 
Danh

**Comment by pandom79** [comment]
I see that the build process fails only for armv7l and aarch64. 
What do i do ?

File: srcpkgs/python3-dlib/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-dlib'
+pkgname=python3-dlib
+version=19.20.0
+revision=1
+wrksrc=dlib-${version}
+build_style=python3-module
+hostmakedepends="cmake python3-setuptools"
+makedepends="python3-devel
+	$(vopt_if openblas openblas-devel)
+	$(vopt_if x11 libX11-devel)"
```
Comment (ericonr) [inline srcpkgs/python3-dlib/template:10]: These tabs should be spaces.

**Comment by ericonr** [comment]
These are probably cross compilation issues. You might have to look into how cmake is invoked from `setup.py`.

**Comment by pandom79** [comment]
Close this pr.

**Pull request closed without merging**